.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,23 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+    - name: npm install, audit, lint and test
+      run: |
+        npm ci
+        npm audit
```

Comment:
@grenewode Is the intent of running `npm audit` to prevent other code from being merged when there are issues found in the audit?

Reply:
The idea with that is to ensure that `npm audit` is being run regularly against the repo, to catch errors in things we have already installed. However, this may not be the best place to do that, since it could result in errors that have nothing to do directly with what individuals have done. Also it would break all pull requests when errors occur

Reply:
So I'd probably be in favor of removing it, now that I'm looking at it again

Reply:
Testing on an old repo of mine, it appears that `npm audit` returns a status code of `1` when a vulnerability is found, which I think would cause the GitHub action to fail.

I think I'm in favor of removing it here for now and finding a way we can run it on a schedule or in some other way that doesn't prevent PRs for unrelated issues.